### PR TITLE
[#5169] Remove Mixpanel

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

**Review by cammellos — APPROVED** [review]

**Review by janherich — APPROVED** [review]

File: src/status_im/data_store/realm/schemas/base/v7/core.cljs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(ns status-im.data-store.realm.schemas.base.v7.core
```
Comment (flexsurfer) [inline src/status_im/data_store/realm/schemas/base/v7/core.cljs:1]: why you didn't add it to `status-im.data-store.realm.schemas.base.core` ?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/data_store/realm/schemas/base/v7/core.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/data_store/realm/schemas/base/v7/core.cljs:1]: fixed

**Review by flexsurfer — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Comment by lukaszfryc** [comment]
@rasom could you update the changelog? Otherwise, it looks good 👍I tested upgrade scenarios, etc. 

But, before merging it, I'd really want to have e2e test results. It maybe that we need to update some user flows there.

**Comment by lukaszfryc** [comment]
We should also add a note to https://wiki.status.im/Help_Improve_Status that we do not collect data using Mixpanel starting from 0.9.23. 

I'll take care of it after PR is merged.

<blockquote><div><strong><a href="https://wiki.status.im/Help_Improve_Status">Help Improve Status - Status Wiki</a></strong></div></blockquote>

**Comment by lukaszfryc** [comment]
@rasom I will update appium tests in scope of this PR

**Comment by rasom** [comment]
@lukaszfryc sure

**Comment by lukaszfryc** [comment]
@yevh-berdnyk I updated appium tests and removed share data screen steps (https://github.com/status-im/status-react/pull/5178/commits/d2f76e0aec78011420b25a8a652e3712777b7339). Could you review it before we squash the commits? 

Also, please check the latest test results. Only `test_token_with_more_than_allowed_decimals` seems to be an issue for me. But, it's probably not related to this PR

**Comment by yevh-berdnyk** [comment]
`test_token_with_more_than_allowed_decimals` - known issue #5171 
`test_network_mismatch_for_send_request_commands` - false failure
`test_transaction_confirmed_on_sender_side` - known issue #4865 
`test_transaction_confirmed_on_recipient_side` - here the test just run out of time waiting for a transaction to be confirmed

**Review by yevh-berdnyk — APPROVED** [review]

**Comment by lukaszfryc** [comment]
Thanks @yevh-berdnyk

**Comment by lukaszfryc** [comment]
@rasom please update the changelog and feel free to merge

**Comment by mandrigin** [comment]
@rasom there is a merge conflict :(

**Comment by rasom** [comment]
on it

**Comment by mandrigin** [comment]
@rasom I don't think our `merge-pr` script is doing well with multiple commits. Can you squash them?

**Comment by rasom** [comment]
@mandrigin i just merged it manually

**Pull request merged**